cmd/analyze/main.go

```diff
@@ -121,7 +121,7 @@ func staticAnalysis(pkg *pkgmanager.Pkg) {
 		sandbox.InitNetwork()
 	}
 
-	sbOpts := makeSandboxOptions(analysis.Static)
+	sbOpts := append(worker.DynamicSandboxOptions(), makeSandboxOptions()...)
```

Comment:
Should be `worker.StaticSandboxOptions()` here

Reply:
Fixed